Cargo.toml

```diff
@@ -141,6 +131,12 @@ members = [
 	"utils/generate-bags",
 ]
 
+default-members = [
+	".", 
+	"bin/execute-worker-cli",
+	"bin/prepare-worker-cli",
```

Comment:
Nice!! Looks better in the root `bin` folder. Just one nit... to me "cli" is not a great name, because it stands for "command line interface" and IMO should apply to programs that the user interfaces with. But these are auxiliary workers and should not be directly interfaced with by users.

Maybe instead, just call them `prepare-worker` etc. and change the libraries to `prepare-worker-lib` etc. WDYT?

Reply:
> `prepare-worker-lib`

Agree, but maybe `prepare-worker-impl` then? They are not libraries in general sense

Reply:
That is better 👍

Reply:
Hmmm, but they do not conflict. I mean, `-impl` is not needed, we can leave `node-core-pvf-execute-worker` as-is, and the binary comes to `polkadot-execute-worker`. Wdyt?

Reply:
It might be confusing if they were named the same, the suffix IMO makes the structure more clear, but no strong opinion here.